Assets/Plugins/Source/Core/LogLevelConfig.cs

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace PlayEveryWare.EpicOnlineServices
+{
+    [Serializable]
+    public class LogCategoryLevelPair
+    {
+        public string category;
+        public string level;
+
+        public LogCategoryLevelPair(string category, string level)
+        {
+            this.category = category;
+            this.level = level;
+        }
+    }
+
+    [Serializable]
+    public class LogLevelConfig : Config
```

Comment:
Because these two classes are only ever utilized by the `LogLevelHelper` class - I suggest that they be made into classes that are nested with in. Furthermore, if I'm not mistaken the first class can also be made to be nested within the second one.

Reply:
This suggestion makes sense to me. It's really limited in its use cases, and would add fewer things to the namespace for people to be concerned with.

Reply:
The reason why they were separated is because the Helper class interacts with the SDK, while the Config only handles pure text/json without knowing the SDK exists

Reply:
Your observation that the Utility class interacts with the SDK while the Config only handles pure text/json is an astute one. I think your reasoning is salient. However - I would argue that making these classes nested does not actually make them interact with the SDK at all - it just limits the scope of where those classes can be utilized.

This is _very_, _very_ pedantic of me (and I take no issue with your reasoning here) but thought I would give voice to my thought-process in case it facilitates more thought on the topic in future circumstances that may be different.